src/common/xo/index.js

```diff
@@ -972,12 +972,19 @@ export const addAcl = ({subject, object, action}) => (
   call('acl.add', resolveIds({subject, object, action}))::tap(
     subscribeAcls.forceRefresh
   )
-)
+).catch(err => error('Add ACL', err.message || String(err)))
```

Comment:
Do you want me to rethrow in all other api methods that do not, even if they are not related to this PR ?...